docs/guides/build-composition.md

```diff
@@ -0,0 +1,279 @@
+
+## Getting Started 
+
+The previous part of the tutorial had you build the first half of your API, the XRD, which defines the shape of your API. Now you are going to build the Composition, which is an implementation of the definition. Open the `composition.yaml` file created earlier. 
+
+Similar to the XRD, it is useful to bootstrap the file by copying in from a “known good” `composition.yaml` and pruning it back to the basics. Using the same repo as before, copy the PostgreSQL [instance composition](https://raw.githubusercontent.com/upbound/platform-ref-gcp/main/package/database/postgres/composition.yaml) and paste it into your `composition.yaml` config. You could also go to the marketplace listing looked at earlier, scroll down and select the “XRDs” tab, and select the Composition titled [xpostgresqlinstances.gcp.platformref.upbound.io](https://marketplace.upbound.io/configurations/upbound/platform-ref-gcp/v0.3.0/compositions/xpostgresqlinstances.gcp.platformref.upbound.io/gcp.platformref.upbound.io/XPostgreSQLInstance), and copy the `.yaml` from there.
+
+{{< img src="../media/basic-compositions/composition-yaml.png" alt="Composition yaml" size="large" >}}
+
+Next, prune out all the fields that are irrelevant to your use case and you will end up with this:
+
+```yaml
+apiVersion: apiextensions.crossplane.io/v1
+kind: Composition
+metadata:
+ name: <todo>
+ labels:
+   provider: <todo>
+spec:
+ writeConnectionSecretsToNamespace: crossplane-system
+ compositeTypeRef:
+   apiVersion: <todo>
+   kind: <todo>
+ resources:
+   - name: <todo>
+     base:
+       apiVersion: <todo>
+       kind: <todo>
+       spec:
+         forProvider:
+     patches:
+```
+
+Next, you will fill in the fields above `resources`:
+
+```yaml
+apiVersion: apiextensions.crossplane.io/v1
+kind: Composition
+metadata:
+ name: xvminstances.compute.acme.co
+ labels:
+   provider: gcp
+spec:
+ writeConnectionSecretsToNamespace: upbound-system
+ compositeTypeRef:
+   apiVersion: compute.acme.co/v1alpha1
+   kind: XVMInstance
+...
+```
+
+Now to explain what you just wrote:
+
+* Under `metadata`, the name convention is to use the composite resource name (which we defined in the XRD).
+* It’s also convention to attach a label of whichever provider this API is built on top of (yours is GCP). This label can be used in `compositionSelector` in an XR or Claim.
+* Each Composition must declare that it is compatible with a particular type of Composite Resource using its `compositeTypeRef` field. The referenced version must be marked `referenceable` in the XRD that defines the XR.
+
+## Composing Managed Resources
+
+This brings you to the main section of the composition, which is where you define the resources that will get composed by the composition and how the fields map (i.e. patch) between the composition and its composed resources. Resources are an array, so you can have anywhere from one to many resources that get composed together. 

```

Comment:
Don't use `i.e.`, say something like "...how the fields map (or patch) between.."